tests/genomics/antismash/test_downloader.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from nplinker.genomics.antismash import download_and_extract_antismash_metadata
+
+class TestDownloadAndExtractAntismashData():
```

Comment:
- I would like to implement a fixture method for use download_root, extract_root, and refseq_assembly_id across the entire class. I tried to look it up online but I don't get how to do it for multiple variables. 
- Apparently, passing in tmp_path creates a default testing dir which is deleted at the end of the test. Is this the best practice though? Would be nice to create temporary folders using `tmpdir` in a fixture method, and then retain the temp folders across the entire class.

Reply:
For `refseq_assembly_id` you can directly use `refseq_assembly_id="GCF_004339725.1"` as a global variable in the class, then you can directly use the variable in functions.

If it's really necessary to share the folders across class, you can see the example using fixture `tmp_path_factory`: https://pytest.org/en/latest/how-to/tmp_path.html#the-tmp-path-factory-fixture

Reply:
I've read the docs there already, but if I try to implement it by creating a temporary folder and yielding it, then the folder is unknown to the other methods of the class.

Reply:
Can the `tmp_dir` be a class attribute? So that the whole class just has one link to that which is created when parsing the class?

Reply:
Indeed, this would be the alternative. But I was looking for using the fixture thing. Anyway it was more for my personal curiosity, it's not needed at all in this specific test :)